grape-entity.gemspec

```diff
@@ -23,7 +23,7 @@ Gem::Specification.new do |s|
 
   s.add_development_dependency 'bundler'
   s.add_development_dependency 'rake'
-  s.add_development_dependency 'rubocop', '~> 0.48'
+  s.add_development_dependency 'rubocop', '~> 0.48.0'
```

Comment:
please also add `require: false`

Reply:
So we need to add the line into `Gemfile` instead of gemspec, I think 🤔 

Could you check https://github.com/ruby-grape/grape-entity/pull/278/commits/8895a69feca402b32dc0fabbc7bc60d93b9e5306 ?

Reply:
And added https://github.com/ruby-grape/grape-entity/pull/278/commits/c362d74460e604cd3a78faf44ef74b267bc7a47f 🙇

Reply:
yhink you can remove this line